packages/ui/src/types/index.ts

```diff
@@ -28,6 +28,11 @@ export type SignInExperienceResponse = Omit<SignInExperience, 'socialSignInConne
   };
 };
 
+// Omit `termsOfUseUrl` since it is being translated into a not null type
+export type SignInExperienceSettings = Omit<SignInExperienceResponse, 'termsOfUseUrl'> & {
+  termsOfUseUrl?: string;
+};
```

Comment:
1. why do we need to translate the value into a no-null type, will there be any issue if we keep it nullabled?
2. why do we need to define a new type for this, instead of directly modifying the original `SignInExperienceResponse` type?

Reply:
updated.